<?xml version="1.0" encoding="utf-8"?>

<launch>
  
  <!-- Start darknet and ros wrapper -->
  <rosparam file="$(find object_management)/config/object_management.yaml"/>
  <node pkg="object_management" type="merge_topics_to_one" name="merge_topics_to_one" output="screen"> 
  
  </node>



 <!--<node name="republish" type="republish" pkg="image_transport" output="screen" 	args="compressed in:=/front_camera/image_raw raw out:=/camera/image_raw" /> -->
</launch>